### PR TITLE
Make swap partitions use RAID0 for RAID tests

### PR DESCRIPTION
**Pull request by OleksandrOrlov**
RAID0 level is required to make all the RAID tests to be consistent and
to allow validation_raid to rely on that consistency, so that the
validation module will not fail if another RAID level is used.

- Related ticket: https://progress.opensuse.org/issues/56870
- Verification run: 
RAID1 (x86_64): https://openqa.suse.de/tests/3446069
RAID10 (ppc64le): https://openqa.suse.de/tests/3446068
RAID6 (aarch64): https://openqa.suse.de/tests/3446070

RAID10 (Tumbleweed-aarch64): https://openqa.opensuse.org/t1050993
RAID5 (Tumbleweed-ppc64le): https://openqa.opensuse.org/t1050382

VR for SLE12: https://openqa.suse.de/tests/3450402
VR for SLE15+: https://openqa.suse.de/t3450405

### CODE REVIEW

File: tests/console/validate_raid.pm [hunk 1]
```diff
@@ -95,14 +93,6 @@ sub prepare_test_data {
...
             $btrfs, $swap,
         );
-        if (is_sle('>=15-SP2') || is_sle('=12-SP5') ||
-            (is_tumbleweed() && !check_var('FLAVOR', 'NET'))) {
-            @raid = ($raid_both_same_level, @raid_detail);
-        }
-        else {
-            # raid0 for swap is required in some products
-            @raid = (($raid_level, $raid0), @raid_detail);
-        }
```
Comment (jknphy) [inline tests/console/validate_raid.pm]: it happen to me as well that when I move the final `else` inside I missed that `@partitioning` and `@raid` do are not declared as default values. So as the code is now, you would miss to test raid on aarch.

Comment (OleksandrOrlov) [reply tests/console/validate_raid.pm]: Ah, ok. I reverted the code to the old state, but didn't notice the issue. Thanks for pointing to it. I've defined `@raid` for aarch64.

Comment (OleksandrOrlov) [reply tests/console/validate_raid.pm]: The new VR: https://openqa.opensuse.org/t1050993

File: tests/console/validate_raid.pm [hunk 2]
```diff
@@ -115,20 +106,13 @@ sub prepare_test_data {
...
-                is_leap() || (is_tumbleweed() && check_var('FLAVOR', 'NET'))) {
-                # raid0 for swap is required in some products
-                @raid = (($raid_level, $raid0), @raid_detail);
-            }
-            else {
-                @raid = ($raid_both_same_level, @raid_detail);
-            }
-        }
+    }
+    else {
```
Comment (jknphy) [inline tests/console/validate_raid.pm]: Besides, you are re-introducing the second false-positive recently fixed by moving `else`, that happens when is `x86_64` and is not SLE <15, for example in openSUSE will not test anything.

Comment (jknphy) [reply tests/console/validate_raid.pm]: Perhaps is a good moment to fill in each `if` `else` with right values for `@raid` and `@partitioning` and die in the last `else` that you re-introduced.

Comment (OleksandrOrlov) [reply tests/console/validate_raid.pm]: I've added `elsif (check_var('ARCH', 'x86_64') && is_sle('<15'))` as for now it looks for me like a simplest solution 

VR for SLE12: https://openqa.suse.de/tests/3450402
VR for SLE15+: https://openqa.suse.de/t3450405

**Pull request merged**